server/vcr-server/api/v2/serializers/rest.py

```diff
@@ -44,6 +44,7 @@ class Meta:
             "first_effective_date",
             "last_effective_date",
         )
+        read_only_fields = fields
```

Comment:
This gives marginal query speed improvements.

https://hakibenita.com/django-rest-framework-slow